### DISCUSSION TITLE
Use card as native wireless modem/ethernet adapter?

### DISCUSSION THREAD

**Discussion by benjamink** [tjboldt/Apple2-IO-RPi#119] [General]
Is it feasible or on the roadmap for consideration to be able to use this as a native wifi modem/network card for the Apple?  It seems like that should be possible but I'm not sure.  

Obviously we can shell into the RPi to run a telnet session or something but would it be possible to run a terminal app on the Apple II & have it use the Apple2-IO-RPi as it's "modem"?

**Comment by tjboldt** [comment]
Yes, that would be possible. Anything where the Apple II is in control and sending commands to the RPi code and getting a response back. So currently that is requesting a block to read or write to the virtual drives, sending characters to a terminal or sending Linux commands.
